src/Services/ClockSettings.vala

```diff
@@ -0,0 +1,22 @@
+/***
+    BEGIN LICENSE
+    Copyright (C) 2016 elementary LLC.
+    This program is free software: you can redistribute it and/or modify it
+    under the terms of the GNU Lesser General Public License version 3, as published
+    by the Free Software Foundation.
+    This program is distributed in the hope that it will be useful, but
+    WITHOUT ANY WARRANTY; without even the implied warranties of
+    MERCHANTABILITY, SATISFACTORY QUALITY, or FITNESS FOR A PARTICULAR
+    PURPOSE.  See the GNU General Public License for more details.
+    You should have received a copy of the GNU General Public License along
+    with this program.  If not, see <http://www.gnu.org/licenses/>
+    END LICENSE
+***/
+public class ClockSettings : Granite.Services.Settings {
```

Comment:
Since we're only going to be fetching one property from this settings page, instead of making a new class just for it you should use GLib.Settings :) It will also allow you to "bind" to the property so changes are updated at creation time

- For creating the Settings class: https://github.com/elementary/wingpanel-indicator-nightlight/blob/master/src/Indicator.vala#L44

- https://github.com/elementary/wingpanel-indicator-nightlight/blob/master/src/Widgets/PopoverWidget.vala#L98

Note: For this to work, the `use_24_format` bool should be changed to a "property" `use24HSFormat { get; set; }`